README.md

```diff
@@ -29,7 +29,7 @@ PowSyBl Open Load Flow provides:
 - An open source implementation of the [SecurityAnalysis API from PowSyBl Core](https://www.powsybl.org/pages/documentation/simulation/securityanalysis/), we support either DC or AC calculations.
 - An open source implementation of the [SensitivityAnalysis API from PowSyBl Core](https://www.powsybl.org/pages/documentation/simulation/sensitivity/), we support either DC or AC calculations.
 
-Almost all of the code is written in Java. It only relies on native code for the [KLU](http://faculty.cse.tamu.edu/davis/suitesparse.html) sparse linear solver. Linux, Windows and MacOS are supported.
+Most of the code is written in Java. It only relies on native code for the [KLU](http://faculty.cse.tamu.edu/davis/suitesparse.html) sparse linear solver. Linux, Windows and MacOS are supported.
```

Comment:
Maybe you should specify the license type of KLU here?

Reply:
@geofjamg I think it is https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/dev/KLU/Doc/License.txt, if we only use this part, right?